GPU/D3D11/StencilBufferD3D11.cpp

```diff
@@ -161,7 +161,7 @@ bool FramebufferManagerD3D11::NotifyStencilUpload(u32 addr, int size, bool skipZ
 	float v1 = 1.0f;
 	MakePixelTexture(src, dstBuffer->format, dstBuffer->fb_stride, dstBuffer->bufferWidth, dstBuffer->bufferHeight, u1, v1);
 	if (dstBuffer->fbo) {
-		draw_->BindFramebufferAsRenderTarget(dstBuffer->fbo, { Draw::RPAction::KEEP, Draw::RPAction::CLEAR });
+		draw_->BindFramebufferAsRenderTarget(dstBuffer->fbo, { Draw::RPAction::KEEP, Draw::RPAction::CLEAR, Draw::RPAction::CLEAR });
```

Comment:
* KEEP, KEEP, CLEAR.  MSDN implies it's possible but I thought there was some reason we were clearing depth here?

-[Unknown]

Reply:
No idea what that was, probably a misunderstanding... I'll make this change in a followup, want to keep this one regression-free.

Reply:
Okay, we did used to only clear stencil on some of these platforms, at least, before BindFramebufferAsRenderTarget.  For some reason I thought this wasn't supported everywhere.

https://github.com/hrydgard/ppsspp/blob/d9acd27126b2f9f7684fdfebbc50d953d1002a77/GPU/GLES/StencilBufferGLES.cpp#L147

https://github.com/hrydgard/ppsspp/blob/d9acd27126b2f9f7684fdfebbc50d953d1002a77/GPU/GLES/StencilBufferGLES.cpp#L197

But yeah, follow up makes sense.

-[Unknown]

Reply:
I also thought so but turns out I was wrong :)